### ISSUE TITLE
Support Fedora

### ISSUE THREAD

**Issue by resynth1943** [WhitewaterFoundry/Fedora-Remix-for-WSL#89]
When purchased from the Microsoft Store, are the resulting funds solely consumed by Whitewater Foundry, Ltd.?
Does this project contribute funds to the upstream vendor: Fedora?

**Comment by crramirez** [comment]
Hello @resynth1943,

Good question. 

We at Whitewater Foundry saw the demand for a Fedora distro for WSL. With no official Fedora for WSL coming, we invested the time and resources into making the Fedora Remix for WSL available as an alternative, building on our premium Pengwin Enterprise codebase. We charge for the official download through the Microsoft Store to continue optimizing, building, and packaging Fedora Remix for WSL, pay open-source developers and build a sustainable open-source venture. 

Fedora Remix for WSL is provided on a community-support basis. There is no support available for Fedora Remix for WSL from the Fedora community other than that which is offered to all users.

In summary, the money that Whitewater Foundry receives for the purchase in Microsoft Store is used to support this project including paying time for developers, this support site, and other expenses related.

Although this project does not directly fund the Fedora Project, it does it indirectly avoiding that Fedora Project invests money in support Fedora on WSL and making it available to the users that is what we do.

Hope it answers your question.

**Comment by resynth1943** [comment]
> Hello @resynth1943,
> 
> Good question.
> 
> We at Whitewater Foundry saw the demand for a Fedora distro for WSL. With no official Fedora for WSL coming, we invested the time and resources into making the Fedora Remix for WSL available as an alternative, building on our premium Pengwin Enterprise codebase. We charge for the official download through the Microsoft Store to continue optimizing, building, and packaging Fedora Remix for WSL, pay open-source developers and build a sustainable open-source venture.
> 
> Fedora Remix for WSL is provided on a community-support basis. There is no support available for Fedora Remix for WSL from the Fedora community other than that which is offered to all users.
> 
> In summary, the money that Whitewater Foundry receives for the purchase in Microsoft Store is used to support this project including paying time for developers, this support site, and other expenses related.
> 
> Although this project does not directly fund the Fedora Project, it does it indirectly avoiding that Fedora Project invests money in support Fedora on WSL and making it available to the users that is what we do.
> 
> Hope it answers your question.

So it's a *no*? Might I ask why? 

Another question: is the  money this project receives publicly available information? I am rather interested in Whitewater Foundry, and would love to learn more about what you do.

I've done some research and appraisal of WSL from Whitewater Foundry is not hard to find. In fact, it's plastered all over your website. Some more research indicates this organisation is not being funded by Microsoft.

> Although this project does not directly fund the Fedora Project, it does it indirectly avoiding that Fedora Project invests money in support Fedora on WSL and making it available to the users that is what we do.

What does this mean?

Thanks for your hasty reply!

**Comment by WSLUser** [comment]
Why would Fedora get funds when Whitewater Foundry had to take take some significant time sinks to make this remix work properly on WSL1? They also managed to identify WSL1 bugs that MS eventually patched. You seem to think this company makes a ton of money but it really doesn't. Not only that but this project releases a free appx here on the github for this distro specifically. The biggest focus of this company is Pengwin. People such as myself also wanted an rpm alternative, hence why this project exists and why it should get funding. The funding helps dedicate time to improving this remix for optimizing the WSL experience. Fedora proper has had legal battles within Red Hat that ended resulting in...no action. WhiteWater Foundry is both a Red Hat partner and MS partner. This gives them freedom to do things for the WSL community that neither MS or Red Hat can do and/or do within reasonable amount of time. If you don't want to support this company's efforts for bettering the WSL experience then don't but please don't make false assumptions about the company.

**Comment by resynth1943** [comment]
> Why would Fedora get funds when Whitewater Foundry had to take some significant time sinks to make this remix work properly on WSL1

Because this software is making money by using the Fedora project, as explained above.

> They also managed to identify WSL1 bugs that MS eventually patched.

Brilliant!

> Not only that but this project releases a free appx here on the github for this distro specifically.

That's very kind of them, but I don't see how this changes the situation.

> You seem to think this company makes a ton of money but it really doesn't.

Please send over any official documentation that supports your claim.

> If you don't want to support this company's efforts for bettering the WSL experience then don't but please don't make false assumptions about the company.

I'm only making assumptions based on what the website is suggesting. As all of their products seem to be orientated around Microsoft, one may conclude this is a Microsoft-funded venture, but apparently that is not the case.

I'd love to hear more, without you whispering these sweet little nothings 😊

**Comment by crramirez** [comment]
Hello @resynth1943,

I will close this issue because first, this is not the place to discuss these kinds of things. If you want company information write to contact@whitewaterfoundry.com. Second, your question was already answered by me and then clarified by @WSLUser, who is not part of the company. We are not a public company, so we don't have the financial data public. And we are not funded by anyone. We only receive money from our customers/users.

Regards,
Carlos
